src/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -0,0 +1,85 @@
+import {
+  isEqual,
+  isString,
+  transform,
+} from "lodash"
+import {
+  optionsMatches,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import shorthandData from "../../reference/shorthandData"
+
+export const ruleName = "declaration-block-no-redundant-longhand-properties"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: (props) => (
+    `Unexpected longhand properties "${props}"`
```

Comment:
I think we've consistently used `Unexpected ...` for rejected messages, especially for `-no-` rules. 

Perhaps that's a needless convention though. In this instance, `Expected shorthand property "border"` certainly seems a more helpful message to the user.